client/__tests__/RatingsAndReviews.test.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
 import { render, screen, cleanup } from '@testing-library/react';
-import RatingsAndReviews from '../src/components/RatingsAndReviews.jsx';
+// import { shallow } from 'enzyme';
```

Comment:
Commented out/Unfinished code should be avoided 95% of the time in industry settings, but for the sake of today's demo, letting this work-in-progress past.

Reply:
Good point. I will put unfinished stuff into another branch going forward. Enzyme is just so cool I wanted to share!